frontend/playwright/ui/pages/login-page.js

```diff
@@ -0,0 +1,27 @@
+class LoginPage {
+  constructor(page) {
+    this.page = page;
+    this.loginButton = page.getByRole("button", { name: "Login" });
+    this.password = page.getByLabel("Password");
+    this.userName = page.getByLabel("Email");
+    this.message = page.getByText("Email or password is incorrect");
+    this.badLoginMsg = page.getByText("Enter a valid email please");
+    this.initialHeading = page.getByRole("heading", { name: "Log into my account" });
+  }
+
+  async url() {
+    this.page.url()
```

Comment:
Is this missing a `return`?

Reply:
ohh I forgot to delete that, it was an error.

Reply:
Thanks, I was not using that function, now it's fixed :)